products/apache-camel.md

```diff
@@ -26,9 +26,16 @@ auto:
 #   Camel may never release patches for non-LTS, but they can still be considered active.
 #   See https://github.com/endoflife-date/endoflife.date/pull/2328#discussion_r1086927567.
 releases:
+-   releaseCycle: "4.6"
+    releaseDate: 2024-05-05
+    eol: false
+    supportedJavaVersions: 17, 21
+    latest: "4.6.0"
+    latestReleaseDate: 2024-05-05
```

Comment:
```suggestion
    releaseDate: 2024-05-10
    eol: false
    supportedJavaVersions: 17, 21
    latest: "4.6.0"
    latestReleaseDate: 2024-05-10
```

According to https://camel.apache.org/blog/2024/05/camel46-whatsnew/, it may be overridden by automation but at least we have the right date in history.